### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by benevenstanciano**
Add TVPWorld.pl

### CODE REVIEW

**Comment by benevenstanciano** [comment]
Had a bunch of silly mistakes on my part but should be fixed now

File: data/channels.csv [hunk 1]
```diff
@@ -23017,7 +23018,8 @@ TVRIMaluku.id,TVRI Maluku,,TVRI,ID,ID-ML,,s/ID-ML;s/ID-MU,ind,,FALSE,,,,http://t
 TVRINasional.id,TVRI Nasional,,,ID,,,c/ID,ind,general,FALSE,1962-08-24,,,http://tvri.go.id/,https://upload.wikimedia.org/wikipedia/commons/thumb/e/eb/TVRILogo2019.svg/512px-TVRILogo2019.svg.png
 TVRINorthKalimantan.id,TVRI North Kalimantan,TVRI Kalimantan Utara,TVRI,ID,ID-KU,,s/ID-KU,ind,,FALSE,,,,http://tvri.go.id/,https://upload.wikimedia.org/wikipedia/commons/thumb/e/eb/TVRILogo2019.svg/512px-TVRILogo2019.svg.png
 TVRINorthSulawesi.id,TVRI North Sulawesi,TVRI Sulawesi Utara,TVRI,ID,ID-SA,,s/ID-SA,ind,,FALSE,,,,http://tvri.go.id/,https://upload.wikimedia.org/wikipedia/commons/thumb/e/eb/TVRILogo2019.svg/512px-TVRILogo2019.svg.png
-TVRINorthSumatra.id,TVRI North Sumatra,TVRI Sumatera Utara,TVRI,ID,ID-SU,,s/ID-SU,ind,,FALSE,,,,http://tvri.go.id/,https://www.lyngsat.com/logo/tv/tt/tvri-sumatera-utara-id.png
+TVRINorthSumatra.id,TVRI North Sumatra,TVRI Sumatera Utara,TVRI,ID,ID-SU,,s/ID-SU,ind,,FALSE,,,,http://tvri.go.id/,https://www.lyngsat.com/logo/tv/tt/tvri-sumatera-utara-id.png
+TVRInfo.ro,TVR Info,,,RO,,,c/RO,ron,news,FALSE,,,,http://stiri.tvr.ro/,https://static.wikia.nocookie.net/logopedia/images/1/19/TVR_Info_2008.svg
```
Comment (LaneSh4d0w) [inline data/channels.csv:23022]:
```suggestion
TVRInfo.ro,TVR Info,,,RO,,,c/RO,ron,news,FALSE,,,,http://stiri.tvr.ro/,https://static.wikia.nocookie.net/logopedia/images/1/19/TVR_Info_2008.svg
```

File: data/channels.csv [hunk 2]
```diff
@@ -22975,7 +22975,8 @@ TVPSeriale.pl,TVP Seriale,,,PL,,,c/PL,pol,series,FALSE,2010-12-06,,,https://seri
 TVPSport.pl,TVP Sport,,,PL,,,c/PL,pol,sports,FALSE,,,,https://sport.tvp.pl/,https://upload.wikimedia.org/wikipedia/commons/thumb/d/d7/Logo_TVP_Sport.jpg/512px-Logo_TVP_Sport.jpg
 TVPublica.ar,TV Pública,,,AR,,,c/AR,spa,general,FALSE,,,,https://www.tvpublica.com.ar/,https://upload.wikimedia.org/wikipedia/commons/0/0a/Logo_Televisi%C3%B3n_P%C3%BAblica_Argentina.png
 TVPublicaFueguina.ar,TV Pública Fueguina,,,AR,AR-V,,s/AR-V,spa,,FALSE,,,,https://tvpublicafueguina.com.ar/,https://i.imgur.com/4dUvFqa.png
-TVPuls.pl,TV Puls,,,PL,,,c/PL,pol,,FALSE,1995-12-01,,,https://tvpuls.pl/,https://upload.wikimedia.org/wikipedia/commons/thumb/d/d6/TV_Puls_logo.jpg/512px-TV_Puls_logo.jpg
+TVPuls.pl,TV Puls,,,PL,,,c/PL,pol,,FALSE,1995-12-01,,,https://tvpuls.pl/,https://upload.wikimedia.org/wikipedia/commons/thumb/d/d6/TV_Puls_logo.jpg/512px-TV_Puls_logo.jpg
+TVPWorld.pl,TVP World,,,PL,,,c/PL,eng,news,FALSE,,,,,https://static.wikia.nocookie.net/logosfake/images/8/8d/TVP_World_%282021%29.svg
```
[the remark this answers is not in the document]
Comment (LaneSh4d0w) [reply data/channels.csv:22979]: Weird downline on these, please check.

File: data/channels.csv (hunk 1 above)
Comment (LaneSh4d0w) [reply data/channels.csv:23022]: Weird downline on these, please check.

**Comment by benevenstanciano** [comment]
Hope that fixes?

**Comment by LaneSh4d0w** [comment]
row 22978 has the wrong line ending character, should be CRLF (data/channels.csv)

**Review by freearhey — APPROVED** [review]

**Comment by freearhey** [comment]
fixed

**Pull request merged**